plugins/services/src/js/routes/services.js

```diff
@@ -171,44 +180,117 @@ const serviceRoutes = [
           },
           {
             type: Route,
+            path: "configuration",
+            title: "Configuration",
+            component: HighOrderServiceConfiguration,
+            buildBreadCrumb() {
```

Comment:
I know, but I didn't want this PR to grow anymore. We have a different task for this already. Hope you don't mind me not including that in this already rather large PR